perception/euclidean_cluster/src/euclidean_cluster_node.hpp

```diff
@@ -20,7 +20,6 @@
 
 #include <geometry_msgs/msg/pose_stamped.hpp>
 #include <sensor_msgs/msg/point_cloud2.hpp>
-#include <tier4_perception_msgs/msg/detected_objects_with_feature.hpp>
```

Comment:
Missing include for new message type.

Reply:
addressed in https://github.com/upowerrobotics/autoware.universe/pull/4/commits/c8ca00935f8de9d8d69f7e31b891090f8aeb396b, the message type headers were added alongside the addition of new conversion function